cockpit/src/subscriptions-client.js

```diff
@@ -274,6 +279,30 @@ client.getSubscriptionStatus = () => {
     });
 };
 
+client.readConfig = () => {
+    return configService.wait(() => configService.GetAll().then(config => {
+        const usingDefaultUrl = (config.server.v.port === '443' &&
+            config.server.v.hostname === 'subscription.rhsm.redhat.com' &&
+            config.server.v.port === '443' &&
+            config.server.v.prefix === '/subscription'
+        );
```

Comment:
Just note: it would be good to have new method of D-Bus: `/com/redhat/RHSM1/Config` (e.g. `IsDefault`) to get information if current configuration value is default or not. This functionality is implemented (look at: `subscription-manager config --list`.